components/web-components/button/index.tsx

```diff
@@ -39,8 +40,8 @@ export function ButtonIconContainer (props: React.PropsWithChildren<{}>) {
   )
 }
 
-export default function Button (props: Props) {
-  const { scale = 'regular' } = props
+export default function Button (props: ButtonProps) {
+  const { scale = 'regular', className } = props
```

Comment:
So my problem here is that it makes our existing button component much less useful. To remove the border, change the background color, or change the text color, I now need to do some form of this:
```css
const ButtonContainer = styled.div`
	&> button {
            --inner-border-size: 0;
            --outer-border-size: 0;
        }
`
because of the way the CSS variables are setup (they are set on the `button` element, so if I just do `--inner-border-size: 0` on the container, it will be overridden by the styles for `Button`. In addition, it seems like wanting to change the padding on the button probably isn't that uncommon, so maybe we should have a `compact` variant, or expose a css variable for controlling padding?

At the moment, (assuming overriding the button styles via a `> button` selector is a no go) it means I can't use the existing `Button` component for any of the buttons in this design (all of them require tweaks either by overriding the `className` or via a `> button` selector) which seems unfortunate.

For now, I've just styled them with the `> button` selector, but happy to go back and add new custom button components.